opus/application/apps/paraminfo/models.py

```diff
@@ -60,6 +61,12 @@ def get_tooltip_results(self):
             definition = get_def_for_tooltip(self.dict_name, self.dict_context)
         return definition
 
+    def get_link_tooltips(self):
```

Comment:
Shouldn't this just be "get_link_tooltip"? There's only one tooltip for this paraminfo.